integration_tests/integration_test.go

```diff
@@ -0,0 +1,39 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+var binPth string
+
+func TestMain(m *testing.M) {
```

Comment:
this is the recommended way to do setup and teardown: https://golang.org/pkg/testing/#hdr-Main